### PR TITLE
align pdf unlocker to native image

### PR DESCRIPTION
**Pull request by GuyAfik**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-5247

## Description
Align the pdf unlocker script to use native-image as well.

### CODE REVIEW

File: Packs/CommonScripts/Scripts/PDFUnlocker/PDFUnlocker.py [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+from PyPDF2 import PdfReader, PdfFileWriter
 
 
 def unlock_pdf(entry_id):
     res = demisto.getFilePath(entry_id)
     origin_path = res['path']
     output_name = "UNLOCKED_" + res['name']
 
-    input1 = PdfFileReader(open(origin_path, "rb"))
+    input1 = PdfReader(open(origin_path, "rb"))
```
Comment (JudahSchwartz) [inline Packs/CommonScripts/Scripts/PDFUnlocker/PDFUnlocker.py:11]: how will this pass on the current 8.1?

Comment (JudahSchwartz) [reply Packs/CommonScripts/Scripts/PDFUnlocker/PDFUnlocker.py:11]: its deprecated, not removed. Why did this fail the build before?

Comment (GuyAfik) [reply Packs/CommonScripts/Scripts/PDFUnlocker/PDFUnlocker.py:11]: https://github.com/py-pdf/pypdf/blob/main/pypdf/_reader.py#L2077 which leads to https://github.com/py-pdf/pypdf/blob/main/pypdf/_utils.py#L384 (Exception).

**Comment by GuyAfik** [comment]
@JudahSchwartz  this PR should be merged only after is merged https://github.com/demisto/content/pull/24056 (there native 8.1 contains the same PyPDF2 library version).

File: Packs/CommonScripts/Scripts/PDFUnlocker/PDFUnlocker.py (hunk 1 above)
Comment (GuyAfik) [reply Packs/CommonScripts/Scripts/PDFUnlocker/PDFUnlocker.py:11]: by the way it seems like it complains about more classes/methods which were deprecated, changed it according to their recommendations?

Comment (JudahSchwartz) [reply Packs/CommonScripts/Scripts/PDFUnlocker/PDFUnlocker.py:11]: you mean about deprecated integrations? They shouldnt be changed

**Review by JudahSchwartz — APPROVED** [review]

File: Packs/CommonScripts/Scripts/PDFUnlocker/PDFUnlocker.py (hunk 1 above)
Comment (GuyAfik) [reply Packs/CommonScripts/Scripts/PDFUnlocker/PDFUnlocker.py:11]: i mean there are several more places which needs to be updated in the PYpdf2 module as well (as you see in the PR)

**Pull request merged**